docs/references/making_release.md

```diff
@@ -0,0 +1,18 @@
+How to make a new release on PyPI and conda-forge
+
+PyPI:
+1. iterate the version number
```

Comment:
we use a system which uses <epochal>.<regular>.<fix>
<epoch> = very large changes. might happen every couple of years. Currently on 1.0
<regular> = regular releases. goal is to do ~3 times per year
<fix> = small patches to fix issues. happen as needed

Reply:
good idea.